applications/asset_tracker/prj.conf

```diff
@@ -56,5 +59,5 @@ CONFIG_NET_IPV4=n
 CONFIG_NET_IPV6=n
 CONFIG_NET_UDP=n
 CONFIG_NET_TCP=n
-CONFIG_NET_RX_STACK_SIZE=256
-CONFIG_NET_TX_STACK_SIZE=256
+CONFIG_NET_RX_STACK_SIZE=512
```

Comment:
I'm curious, did anything crash for you so that you needed to increase these stacks?

Reply:
Oops, this shouldn't be here. Will remove it.